internal/clients/ec2/ec2_client.go

```diff
@@ -99,3 +99,16 @@ func (c *Client) CreateEC2ClientFromConfig(crd *stsTypes.Credentials) (*Client,
 
 	return newClient, nil
 }
+
+func (c *Client) ListInstanceTypes() ([]types.InstanceTypeInfo, error) {
+	log.Trace().Msg("Listing AWS EC2 instance types")
+	input := &ec2.DescribeInstanceTypesInput{
+		MaxResults: aws.Int32(100),
```

Comment:
For reference: 
https://github.com/RHEnVision/provisioning-backend/pull/66/commits/83a5454dd691c07f26c66180c76637dd86aced2c#r907188833